app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\UserUpdateRequest;
+use App\Models\User;
+use App\Services\UserService;
+use Symfony\Component\HttpFoundation\JsonResponse;
+
+class UserController extends Controller
+{
+    public function __construct(UserService $userService)
+    {
+        $this->userService = $userService;
+    }
+
+    public function getUsers(): JsonResponse
+    {
+        $users = User::all();
+
+        return response()->json([
+            'users' => $users,
+            'message' => 'Users found',
+            'status' => 1
+        ]);
+    }
+
+
+    public function update(UserUpdateRequest $request)
```

Comment:
your controller action also must get a user id to update.
```
public function update(UserUpdateRequest $request, int $userId)
{}
```

or you can investigate model binding [https://laravel.com/docs/10.x/routing#route-model-binding](click) and use it instead:
`public function update(UserUpdateRequest $request, User $user)
{}`